bcda/monitoring/monitoring.go

```diff
@@ -35,13 +32,15 @@ func (a apm) End(txn *newrelic.Transaction) {
 func GetMonitor() *apm {
 	if a == nil {
 		target := conf.GetEnv("DEPLOYMENT_TARGET")
+		enabled := true
```

Comment:
added this section because then it won't send metrics to new relic during CI runs.